src/plugins/synced-lyrics/providers/LRCLib.ts

```diff
@@ -49,6 +49,7 @@ export class LRCLib implements LyricProvider {
       const trackName = alternativeTitle || title;
       query = new URLSearchParams({ q: `${trackName}` });
       url = `${this.baseUrl}/api/search?${query.toString()}`;
+      artist = trackName.split(' - ')[0];
```

Comment:
YouTube has many edge cases, so it's not appropriate to parse the title and artist name using such a simple approach.